app-editors/diakonos/diakonos-0.9.8.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+USE_RUBY="ruby27 ruby30 ruby31"
+
+inherit ruby-ng
+
+DESCRIPTION="A Linux editor for the masses"
+HOMEPAGE="https://git.sr.ht/~pistos/diakonos"
+SRC_URI="https://git.sr.ht/~pistos/diakonos/archive/${206b112ac989457b8e51b95c272c380f4bf9221bPV}.tar.gz -> ${P}.tar.gz"
```

Comment:
The commit you posted seems does not exist. The right way to do so is to use COMMIT="abcxyz"
SRC_URI="https://https://git.sr.ht/~pistos/diakonos/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
That's because at 90% of times you need ${COMMIT} for ${S} too. On top of that I don't see any 0.9.8 releases around the site you have posted... did you assume it will be released as it? I'd suggest to use diakonos-0.9.7_p20230706 instead, if you mind using using commit 8493d9a91d936b22eaded08fbbea849db2e96921

Reply:
> https://git.sr.ht/~pistos/diakonos/commit/5f064a5d439b545d7eea360caef2b40114dd294b Here's the 0.9.8 release

Well, it should be better to superseed the commit and bump to 0.9.12 https://git.sr.ht/~pistos/diakonos/commit/882251ed8f0406973e80d67956fd6a2d5c7fc923